### PR TITLE
Add rector on block bundle

### PR DESCRIPTION
**Pull request by jordisala1991**
Related: https://github.com/sonata-project/SonataBlockBundle/pull/1057

### CODE REVIEW

**Review by VincentLanglet — APPROVED** [review]

File: templates/project/.github/workflows/qa.yaml.twig [hunk 1]
```diff
@@ -70,7 +70,6 @@ jobs:
             - name: Psalm
               run: vendor/bin/psalm --no-progress --show-info=false --stats --output-format=github --threads=$(nproc) --shepherd --php-version={{ branch.targetPhpVersion.toString }}
 {% if project.rector %}
-
```
Comment (VincentLanglet) [inline templates/project/.github/workflows/qa.yaml.twig:73]: We add a line between every step. Might be better to keep this line and add one before
```
psalm:
```

WDYT ?

Comment (jordisala1991) [reply templates/project/.github/workflows/qa.yaml.twig:73]: Yes we can do that instead.

File: templates/project/.github/workflows/qa.yaml.twig [hunk 2]
```diff
@@ -70,7 +70,6 @@ jobs:
             - name: Psalm
               run: vendor/bin/psalm --no-progress --show-info=false --stats --output-format=github --threads=$(nproc) --shepherd --php-version={{ branch.targetPhpVersion.toString }}
 {% if project.rector %}
```
Comment (jordisala1991) [inline templates/project/.github/workflows/qa.yaml.twig:72]:
```suggestion
{% if project.rector %}

```

File: templates/project/.github/workflows/qa.yaml.twig (hunk 1 above)
Comment (jordisala1991) [reply templates/project/.github/workflows/qa.yaml.twig]: Lets only enable rector for block bundle for now. Later we will fix this empty line issue

**Pull request merged**